components/brave_ads/rust/BUILD.gn

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2023 The Brave Authors. All rights reserved.
```

Comment:
The convention is to name the directory `rs` (instead of rust) and the the rust src should go in `rs/src`.